BikeDisplaySite.page

```diff
@@ -0,0 +1,77 @@
+<apex:page standardController="Lead" extensions="BikeDisplayController">
+
+<style type="text/css">
+  .PageBlock {                
+         font-weight:bold;        
+        }
+    </style>
+    
+<apex:form id="bi">
+<apex:pageblock title="Bike Designs" id="PageBlock">
```

Comment:
This is hardcoded and limited to 5 only. It is suppose to be dynamic based on number of bikes retrived from database.